myhpi/core/models.py

```diff
@@ -107,6 +107,21 @@ class Minutes(Page):
     parent_page_types = ["MinutesList"]
     subpage_types = []
 
+    @property
+    def next_minutes(self):
+        siblings = self.get_next_siblings().live()
+        # TODO: When #57 is merged, iterate through pages and check if user can view them
+        if siblings:
+            return siblings[0]
```

Comment:
what is the order here? will the first item in the list always be the minutes that we are looking for? or is this influenced by time of last edit or similar?

Reply:
I think it is creation time (the order of pages in the tree)

Reply:
This may sometimes be the wrong order, e.g. because minutes of GF-Treffen are written by hand and digitized afterwards. There may have been another meeting in between.